engines/dfc_provider/app/controllers/dfc_provider/catalog_items_controller.rb

```diff
@@ -8,6 +8,12 @@ class CatalogItemsController < DfcProvider::ApplicationController
 
     def index
       person = PersonBuilder.person(current_user)
+
+      enterprises = current_user.enterprises.map do |enterprise|
+        EnterpriseBuilder.enterprise(enterprise)
+      end
+      person.affiliatedOrganizations = enterprises
+
       render json: DfcIo.export(
         person,
```

Comment:
I don't think I understand why we need to output the user here, but I guess it makes it explicit that we're only outputting catalog items associated with the authenticated user.

Reply:
In other APIs it's implicit what the resulting data set is representing. You query catalog items and in one app you get _your_ catalog items and in another app you may get _all_ catalog items (but then whose?). DFC is always making this connection explicit, telling you how it fits into the graph of data. Any Semantic Web web can store the information in the right place without even knowing your app's logic.

Reply:
Thanks for explaining, that makes total sense!